substrate/frame/assets/src/tests.rs

```diff
@@ -1775,3 +1775,77 @@ fn asset_destroy_refund_existence_deposit() {
 		assert_eq!(Balances::reserved_balance(&admin), 0);
 	});
 }
+
+#[test]
+fn transfer_and_preserve_account() {
+	new_test_ext().execute_with(|| {
+		use frame_support::traits::tokens::Preservation::Preserve;
+
+		let id = 42;
+		Balances::make_free_balance_be(&1, 10);
+		Balances::make_free_balance_be(&2, 10);
+		assert_ok!(Assets::create(RuntimeOrigin::signed(1), id, 1, 10));
```

Comment:
Is calling `create` necessary? I generally just call `mint` directly on uninitialised accounts.